src/countdown360.js

```diff
@@ -8,9 +8,14 @@ class Countdown360 extends React.Component {
   constructor (props) {
     super(props)
     const seconds = Math.max(props.seconds, 0)
+    const secondsLeft =
+      props.remainingSeconds && props.remainingSeconds > 0
+        ? Math.min(seconds, props.remainingSeconds)
+        : seconds
+
     this.state = {
       totalSeconds: seconds,
-      secondsLeft: seconds * 1000,
+      secondsLeft: secondsLeft * 1000,
```

Comment:
This reminds me that I should rename this variable to "millisecondsLeft"... I'll do this myself, don't worry about it!